docs/database.md

```diff
@@ -0,0 +1,66 @@
+# Database Servers (MySQL)
+
+The primary data store for user content is the database server.
+
+Altis uses MySQL-compatible servers running on [Amazon Aurora](https://aws.amazon.com/rds/aurora/), with an auto-scaling, replicated configuration.
+
+
+## Multi-Server Configuration
+
+Unlike many WordPress hosts, Altis Cloud environments use a multi-server configuration for the database, with fully dedicated servers for your database. Servers are categorised into two general groups:
+
+* **Primary**: The primary server is the authoritative server storing all data.
+* **Replica:** The replica servers store copies of the data, continually synchronised from the primary.
+
+All database writes (`INSERT`, `UPDATE`, `DELETE`, etc) are sent directly to the primary server, while database reads (`SELECT`) are sent to the replicas. This allows offloading most of the database load from the primary.
```

Comment:
`SELECT`s can be sent to either writers or readers, might not be important though.